connector/connect/common/src/main/buf.gen.yaml

```diff
@@ -16,18 +16,18 @@
 #
 version: v1
 plugins:
-  - remote: buf.build/protocolbuffers/plugins/cpp:v3.20.0-1
+  - plugin: buf.build/protocolbuffers/cpp:v21.7
     out: gen/proto/cpp
-  - remote: buf.build/protocolbuffers/plugins/csharp:v3.20.0-1
+  - plugin: buf.build/protocolbuffers/csharp:v21.7
     out: gen/proto/csharp
-  - remote: buf.build/protocolbuffers/plugins/java:v3.20.0-1
+  - plugin: buf.build/protocolbuffers/java:v21.7
     out: gen/proto/java
   - plugin: buf.build/grpc/ruby:v1.56.0
     out: gen/proto/ruby
-  - remote: buf.build/protocolbuffers/plugins/ruby:v21.2.0-1
+  - plugin: buf.build/protocolbuffers/ruby:v21.7
     out: gen/proto/ruby
    # Building the Python build and building the mypy interfaces.
-  - remote: buf.build/protocolbuffers/plugins/python:v3.19.3-1
+  - plugin: buf.build/protocolbuffers/python:v21.7
```

Comment:
https://buf.build/protocolbuffers/python
plugin: buf.build/protocolbuffers/python:v23.4
It seems that this version is the latest.